articles/active-directory/develop/msal-net-initializing-client-applications.md

```diff
@@ -97,7 +97,7 @@ The modifiers you can set on a public client or confidential client application
 
 |Modifier | Description|
 |--------- | --------- |
-|`.WithAuthority()` 7 overrides | Sets the application default authority to an Azure AD authority, with the possibility of choosing the Azure Cloud, the audience, the tenant (tenant ID or domain name), or providing directly the authority URI.|
+|`.WithAuthority()`  | Sets the application default authority to an Azure AD authority, with the possibility of choosing the Azure Cloud, the audience, the tenant (tenant ID or domain name), or providing directly the authority URI.|
```

Comment:
Looks like their intent was to say that the [WithAuthoriy()](https://docs.microsoft.com/dotnet/api/microsoft.identity.client.abstractapplicationbuilder-1.withauthority?view=azure-dotnet) method has seven overrides, each taking different parameters.

We really shouldn't be duplicating API reference in our conceptual reference - that's how we end up with confusion like this.

How about:

```suggestion
|`[.WithAuthority()](/dotnet/api/microsoft.identity.client.abstractapplicationbuilder-1.withauthority)`  | Sets the application default authority to an Azure AD authority, with the possibility of choosing the Azure Cloud, the audience, the tenant (tenant ID or domain name), or providing directly the authority URI.|
```